integration/host_vulnerability_test.go

```diff
@@ -367,3 +410,37 @@ func validRFC3339(field string, jsonString string) bool {
 	}
 	return true
 }
+
+func machineIDExists(lw *api.Client) bool {
+	resp, err := lw.Vulnerabilities.Host.GetHostAssessment(hostname)
```

Comment:
This function doesn't lookup for a hostname, but rather an ID.
```suggestion
	resp, err := lw.Vulnerabilities.Host.GetHostAssessment(machineID)
```